lmdeploy/pytorch/backends/cuda/op_backend.py

```diff
@@ -175,6 +175,17 @@ def build_graph_runner(model: torch.nn.Module, model_config: ModelConfig, cache_
                            backend_config: BackendConfig, device: torch.device):
         """build graph runner."""
         from .graph_runner import CUDAGraphRunner
+        from .warmup_manager import WarmupMeta, get_warmup_manager
+
+        # warmup ops.
+        warmup_meta = WarmupMeta(
```

Comment:
May we consider to use [NVRTC](https://docs.nvidia.com/cuda/nvrtc/index.html) to speedup

Reply:
Sounds cool